infrastructure/main.tf

```diff
@@ -232,6 +233,7 @@ locals {
     MICROSERVICEKEYS_COH_COR                    = "${data.vault_generic_secret.cohcor.data["value"]}"
     MICROSERVICEKEYS_BULK_SCAN_PROCESSOR        = "${data.vault_generic_secret.bulkScanProcessor.data["value"]}"
     MICROSERVICEKEYS_BULK_SCAN_PROCESSOR_TESTS  = "${data.vault_generic_secret.bulkScanProcessorTests.data["value"]}"
+    MICROSERVICEKEYS_BULK_SCAN_ORCHESTRATOR     = "${local.microservice_key_names["BULK_SCAN_ORCHESTRATOR"]}"
```

Comment:
it has to be the secret here, so `local.microservice_key_settings[...`

Reply:
fixed